rocketpool-cli/service/service.go

```diff
@@ -1055,6 +1043,16 @@ func pruneExecutionClient(c *cli.Context) error {
 
 	fmt.Printf("Your disk has %s free, which is enough to prune.\n", freeSpaceHuman)
 
+	if selectedEc == cfgtypes.ExecutionClient_Nethermind {
+		// Restarting NM is not needed anymore
+		err = rp.RunNethermindPruneStarter(executionContainerName)
+		if err != nil {
+			return fmt.Errorf("Error starting Nethermind prune starter: %w", err)
+		}
+		fmt.Printf("\nDone! Your main execution client is now pruning. You can follow its progress with `rocketpool service logs eth1`.\n")
```

Comment:
split this into:
```golang
fmt.Println()
fmt.Println("Your main execution client is now pruning. You can follow its progress with `rocketpool service logs eth1`.")
```

there's no need for printf with a single string, and separate printlns makes the code read closer to the output, which improves readability